components/ide/jetbrains/backend-plugin/build.gradle-stable.kts

```diff
@@ -72,6 +87,20 @@ dependencies {
     implementation("io.grpc:grpc-netty-shaded:1.49.0")
 }
 
+tasks.withType<KotlinCompile> {
+    if (properties("platformType") == "RD") {
+        print("Rider: exclude unnecessary files")
+        exclude("**/GitpodForceUpdateMavenProjectsActivity.kt")
+        exclude("**/maven.xml")
+    }
+}
+
+tasks.named("test") {
+    onlyIf {
+        properties("platformType") != "RD"
+    }
+}
```

Comment:
Was this being used earlier for testing? For example, to execute code when Platform Type wasn't Rider? I'm not sure of it's value "now".